packages/web-config-server/src/apiV1/dataBuilders/generic/percentage/percentagesOfValueCountsPerPeriod.js

```diff
@@ -16,7 +16,7 @@ const filterFacility = async (models, filterCriteria, analytics) => {
   const facilities = await models.facility.find({
     type: {
       comparator: filterCriteria.comparator,
-      comparisonValue: '1',
+      comparisonValue: '1%',
```

Comment:
Should work as long as we don't add more than 9 facility types. :thinking:

Reply:
oh yeah @nick-beyondessential that's a good point. Not sure if that works with Tupaia but I will try `"type" similar to '(1|1.%)'`, which should handle more than 9 facility types.

Reply:
I wouldn't stress too much about it - not sure how likely it is.

Reply:
Will keep that in mind.